src/server/pbsd_init.c

```diff
@@ -702,9 +696,15 @@ pbsd_init(int type)
 		}
 
 		/* now do sched db */
-
-		if ((rc = sched_recov_db()) == -1) {
+		rc = sched_recov_db();
+		if (rc == -1) {
 			log_err(rc, "pbsd_init", "unable to recover scheddb");
+		} else if(rc == -2) {
+			/* No Schedulers found in DB */
+			/* Create and save default to DB*/
+			dflt_scheduler = sched_alloc(PBS_DFLT_SCHED_NAME);
+			(void)sched_save_db(dflt_scheduler, SVR_SAVE_NEW);
+			set_sched_default(dflt_scheduler);
```

Comment:
but why not? you are taking the trouble of making default attribs every now and then, instead it would have been easy if they were just stored in DB.